### ISSUE TITLE
Pressable does not accept aria-disabled

### ISSUE THREAD

**Issue by leononame** [necolas/react-native-web#2653]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the issue

When building a button using `Pressable`, the `aria-disabled` prop is not actually forwarded into the DOM.

### Expected behavior

Both the `div` and the `button` element in the linked sample should have `aria-disabled` set.

### Steps to reproduce

rnw 0.19.10, all browsers

### Test case

https://codesandbox.io/p/sandbox/tender-khorana-y3zgl6

**Comment by necolas** [comment]
`disabled` maps to `aria-disabled` https://github.com/necolas/react-native-web/blob/master/packages/react-native-web/src/exports/Pressable/index.js#L204

**Comment by leononame** [comment]
Ok, understood! Thanks.
